editors/README.md

```diff
@@ -165,16 +166,26 @@ the name of the component you would like to preview and select the "Show preview
 will appear on the right of the editor pane.
 
 ## JetBrains IDE
-Tested with: CLion 2022.1 (on linux)
 
-Tested and doesn't work with CLion 2023.1+ (on Linux). Plugin is no longer maintained.
+https://github.com/kizeevov/slint-idea-plugin has a plugin for the Intellij
+platform.
+
+_Note: This plugin is developed by @kizeevov._
+
+### "LSP Support" plugin (unmaintained)
 
-*Note: This may/should work for other JetBrains-products also. If you test it please add your results to the above list.*
+CLion 2022.1 (on linux) had an optional "LSP Support" plugin, which could be
+configured to work with the `slint-lsp`. Unfortunately we got reports
+that CLion 2023.1+ (on Linux) no longer works with that plugin and that development
+has stopped.
+
+This is how the plugin used to be set up:
 
 ### LSP Support
```

Comment:
This headings refer to the old unmaintained plugin so it should be under it. Or removed.

But don't you think we should remove this entire paragraph if it is not working anyway?